README.md

```diff
@@ -1,6 +1,89 @@
-# JanusGraph Python driver
+# JanusGraph-Python
 
-## License
+## Python Client drivers for [JanusGraph](http://janusgraph.org)
+
+JanusGraph-Python is the Python drivers for connecting to JanusGraph. 
+It extends Apache TinkerPop™'s [Gremlin-Python](http://tinkerpop.apache.org/docs/current/reference/#_gremlin_python) 
```

Comment:
The link is broken. It should be:
http://tinkerpop.apache.org/docs/current/reference/#gremlin-python